_data/simple-icons.json

```diff
@@ -14790,7 +14790,7 @@
         {
             "title": "TinyLetter",
             "hex": "ED1C24",
-            "source": "https://tinyletter.com/site/press/"
+            "source": "https://tinyletter.com/site/press"
         },
```

Comment:
As stated in https://discord.com/channels/1142044630909726760/1151132343776591899/1151169674113978411, removing trailing slashes from path does not always work. You can try it with this one. While https://tinyletter.com/site/press/ is working, https://tinyletter.com/site/press is not.

If we want to do this, we'll need to check them all!

Reply:
Well, feck! 😕 What sort of lunatic sets a web server up like that these days?!

Given that, I think we're better off closing this PR and tackling each one individually as part of the Great Icon Review/